hack/bazel.sh

```diff
@@ -20,7 +20,10 @@ set -o errexit
 set -o pipefail
 
 code=0
-(set -o xtrace && bazel "$@") || code=$?
+(set -o xtrace && bazel run //prow/test/integration:setup) || code=$?
```

Comment:
without RBE this step takes ~6 minutes, will need to think about how to pass in `--config=ci` from prow job.

Reply:
Try grabbing --config=<> from args